web/src/components/PlayerCard/PlayerCardFb.jsx

```diff
@@ -70,17 +70,30 @@ const PlayerCardFb = ({ sourceId, setVol }) => {
                     </StopProp>
                 </div>
                 <div className="content">
-                    { !is_streamer && (
-                        <div className="zones">
-                          <ZonesBadge sourceId={sourceId} onClick={openZones} />
-                        </div>
-                    )}
-                    { is_streamer && (
-                        <div className="streamer-outputs">
-                            <StreamerOutputBadge sourceId={sourceId} />
-                        </div>
-                    )}
-                    <SongInfoMarquee sourceId={sourceId} />
+                    <Grid
+                        container
+                        direction="row"
+                        justifyContent="center"
+                        alignItems="center"
+                    >
+                        <Grid item xs={4} md={3}>
+                            { !is_streamer && (
+                                <div className="zones">
+                                  <ZonesBadge sourceId={sourceId} onClick={openZones} />
+                                </div>
+                            )}
+                            { is_streamer && (
+                                <div className="streamer-outputs">
+                                    <StreamerOutputBadge sourceId={sourceId} />
+                                </div>
+                            )}
+                        </Grid>
+                        <Grid item xs={4} md={6}> {/* Spacer */} </Grid>
+                        <Grid item xs={4} md={3}>
+                            <SongInfo sourceId={sourceId}/>
+                        </Grid>
+
+                    </Grid>
```

Comment:
I've added support for the MUI Grid component, which is a fast way of implementing CSS grid formatting support. It has built in breakpoints (xs, sm, md, lg, xl) for different screen sizes, then you identify what relative size you want everything to be, interpreted as a percentage of adding all the same breakpoints up together

I always have my breakpoints add up to 12 as that's the tutorial on the mui website, though that isn't necessary. In this case, at the xs breakpoint everything is sized at 4/12 (33%) and at md and larger it's sized [3/12, 6/12, 3/12]  [25%, 50%, 25%]

The primary reason I went to grid formatting is this:

Old:
![image](https://github.com/micro-nova/AmpliPi/assets/114415937/daeef19b-d789-400e-a498-1e86d780f951)


New:
![image](https://github.com/micro-nova/AmpliPi/assets/114415937/66360ead-67b7-4a19-a1ff-9b198e7e32e9)

Reply:
I've since changed these particular breakpoints a little, but the underlying explanation is still good for those who wish to know how the Grid breakpoints work